buildtest/config.py

```diff
@@ -54,34 +54,34 @@ def check_configuration(config_path=None):
 
     logger = logging.getLogger(__name__)
 
-    user_schema = load_configuration(config_path)
+    user_schema = load_settings(config_path)
 
-    config_schema = load_schema(DEFAULT_CONFIG_SCHEMA)
-    logger.debug(f"Loading default configuration schema: {DEFAULT_CONFIG_SCHEMA}")
+    config_schema = load_schema(DEFAULT_SETTINGS_SCHEMA)
+    logger.debug(f"Loading default configuration schema: {DEFAULT_SETTINGS_SCHEMA}")
 
     logger.debug(f"Validating user schema: {user_schema} with schema: {config_schema}")
     validate(instance=user_schema, schema=config_schema)
     logger.debug("Validation was successful")
 
 
-def load_configuration(config_path=None):
-    """Load the default configuration file if no argument is specified.
+def load_settings(config_path=None):
+    """Load the default settings file if no argument is specified.
```

Comment:
And here as well.

Reply:
i replaced all ``config_paths`` with ``settings_path`` in ``buildtest/config.py``. Yes there are more instance i see. Good catch